src/components/ArticlesList.js

```diff
@@ -1,52 +1,70 @@
 import React, { useEffect, useState } from 'react';
 import { getArticles } from '../api';
 import '../styling/articlesList.css';
+import '../styling/articleCard.css';
 import ArticleCard from './ArticleCard';
+import { Link } from 'react-router-dom';
 
 function ArticlesList() {
   const [articles, setArticles] = useState([]);
   const [articlesLoading, setArticlesLoading] = useState(true);
   const [page, setPage] = useState(1);
+  const [maxPage, setMaxPage] = useState(1);
 
   useEffect(() => {
-    getArticles(page).then((articles) => {
+    setArticlesLoading(true);
+    getArticles(page).then(({ articles, total_count }) => {
       setArticles(articles);
+      setMaxPage(Math.ceil(total_count / 10));
       setArticlesLoading(false);
     });
   }, [page]);
 
   const nextHandler = () => {
     setPage((currPage) => {
-      return currPage + 1;
+      return ++currPage;
     });
   };
 
   const prevHandler = () => {
     setPage((currPage) => {
-      return currPage - 1;
+      return --currPage;
     });
   };
```

Comment:
you could refactor these to be one function that sets page to either curr+1 or curr-1